apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -48,7 +48,9 @@
     quill.setContents(quill.clipboard.convert({ html: contents }));
 
     quill.on('text-change', function () {
-      let contents = rtePurify.sanitize(quill.getSemanticHTML(), rtePurifyConfig);
+      let contents = quill.editor?.isBlank?.()
```

Comment:
I opened https://github.com/slab/quill/issues/4254 to see if they'd be willing to commit to this as part of the public API. That said, I'd like to see the reason for the optional chaining documented here, probably with a link to that issue.

IMO it'd also be nice to inline some of the context from your PR:

- Why we need the check for `isBlank()`
- Why we can't use `getText().trim() === ''`
- The caveat that this won't do quite what's expected if e.g. someone starts a bulleted list but doesn't put any text in the list items

Reply:
Done in b3ed477a7.